### PR TITLE
API endpoint for subscribers

### PR DESCRIPTION
**Pull request by ethantkoenig**
Add the following endpoint:
- `GET /repos/:owner/:reponame/subscribers`: list a repo's subscribers/watchers

### CODE REVIEW

File: routers/api/v1/repo/subscriber.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package repo
```
Comment (lunny) [inline routers/api/v1/repo/subscriber.go:5]: copyright header

**Comment by lunny** [comment]
LGTM except copyright header

**Comment by ethantkoenig** [comment]
Copyright added

**Comment by lunny** [comment]
conflict

**Comment by ethantkoenig** [comment]
fixed

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Pull request merged**